cmd/cephcsi.go

```diff
@@ -101,6 +104,11 @@ func getDriverName() string {
 }
 
 func main() {
+	if conf.Version {
+		fmt.Println("cephcsi Version:", util.DriverVersion)
```

Comment:
Could you include the util.GitCommit too please? Just on an additional line would be fine, make it easy to parse in a script.

Reply:
```
Cephcsi Version: v1.2.1
Git Commit: 4b871366327d63e27fc1abfb699f0faaf0fc16b9
GoVersion: go1.12.5
Compiler: gc
Platform: linux/amd64
```
added some more information, PTAL